### PR TITLE
RUn2-TB28 Add TB dependent module making algorithm to introduce rectangular absorber

### PR DESCRIPTION
**Pull request by bsunanda**
The absorber layers are divided into 2 parts: one as part of the module, other rectangular shapes. They are applicable to 2017 TB

### CODE REVIEW

**Comment by ianna** [comment]
please test

**Comment by ianna** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

**Comment by kpedro88** [comment]
+1

File: Geometry/HGCalCommonData/plugins/DDHGCalTBModule.h [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+protected:
+
+  void          constructLayers (DDLogicalPart, DDCompactView& cpv);
+  double        rMax(double z);
+  void          positionSensitive(DDLogicalPart& glog, int type, double rin,
+				  double rout, DDCompactView& cpv);
+
+private:
+
+  std::vector<std::string> wafer;         //Wafers
```
Comment (davidlange6) [inline Geometry/HGCalCommonData/plugins/DDHGCalTBModule.h]: hi @bsunanda - please follow a naming convention for class member variables (eg, in CMS its often blah_)
Thanks

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@ianna @kpedro88 I just changed variable names as suggested by David. Could you approve this PR?

**Comment by kpedro88** [comment]
+1

**Comment by ianna** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**